### ISSUE TITLE
API only installation

### ISSUE THREAD

**Issue by ihabassi** [Wildhoney/Magento-on-Angular#85]
Hi,
Is it possible to install the API only? If yes do you have any step by step instruction how to install it.

Thanks,
Ihab

**Comment by Cool-and-Deadly** [comment]
@aonic Does that mean to retrieve product data via API (and not via sockets) you simply leave out the command `node node/server.js`?

I tried it and without the node command I can't retrieve any products. The browser console outputs an Angular digest error:
`Uncaught Error: [$rootScope:infdig] 10 $digest() iterations reached. Aborting!`

Any clue?

**Comment by Wildhoney** [comment]
You need to modify the code to retrieve the products from `/products`, rather than relying on WebSockets.

**Comment by aonic** [comment]
Yes, I thought you wanted to use the API server without the Angular front-end (in a different project). You can do that, but the Angular code as-is uses the web socket from the node server

**Comment by Cool-and-Deadly** [comment]
OK sorry I think I misunderstood .. 
@aonic you're right @ihabassi might just wanna use the API in a different context.
